test/fixtures/inject/valid/package.json

```diff
@@ -59,17 +55,22 @@
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-react-hooks": "^4.5.0",
     "prettier": "^2.6.2",
-    "react": "^18.0.0",
+    "prettier-plugin-packagejson": "^2.3.0",
+    "react": "^18.2.0",
     "rimraf": "^3.0.2",
-    "sanity": "dev-preview || 3.0.0-rc.0",
+    "sanity": "3.0.0-rc.0",
     "typescript": "^4.7.3"
   },
   "peerDependencies": {
-    "react": "^17.0.0 || ^18.0.0",
+    "react": "^18",
```

Comment:
Absolutely super that you updated these, but dependencies are mostly not used in tests. To be fair, we should probably strip away a lot of stuff so only the relevant things for thes test are in the package.json. But I was a lazy boy 🙈 Much better now :D